src/shared/components/common/error-guard.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component } from "inferno";
+import { setIsoData } from "../../utils";
+import { ErrorPage } from "../app/error-page";
+
+class ErrorGuard extends Component<any, any> {
+  private isoData = setIsoData(this.context);
+
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+
+  render() {
+    const errorPageData = this.isoData.errorPageData;
+    const siteRes = this.isoData.site_res;
+    console.log("In error guard");
+    console.log(errorPageData);
```

Comment:
Needs to remove the auth data, somewhere in this file most likely.

Reply:
The error only gets added to the page data when the UI is run in development. From lines 300-304 in the server `index.ts`:
```typescript
  // Exact error should only be seen in a development environment. Users
  // in production will get a more generic message.
  if (process.env.NODE_ENV === "development") {
    errorPageData.error = error;
  }
```

However, I will remove the console logs as they are unnecessary.